Modify rule S4624: Adapt to LaYC (#2399)
SonarSource/rspec: rules/S4624/javascript/rule.adoc

== Why is this an issue?

Template literals, also known as template strings, allow for string interpolation and multiline strings in JavaScript. They provide a more convenient and flexible way to work with strings compared to traditional string concatenation or manipulation.

> Template literals are a convenient way to include variables or expressions within a string by using the backtick ``++(`)++`` character. They allow you to embed placeholders ``++`${expression}`++`` in the string and evaluate them dynamically.

Reviewer: ```suggestion
Template literals are delimited with the backtick ``++(`)++`` character. They are a convenient way to include variables or expressions within a string using placeholders ``++`${expression}`++`` in the string and evaluate them dynamically.
```

Reviewer: in the first phrase seems you imply that you can use the backtick to include variables. I would separate the backtick syntax from the placeholder one.
